aip/0002.md

```diff
@@ -44,16 +44,30 @@ Currently recognized blocks of AIP numbers are:
 - **1-99:** Reserved for meta-AIPs (generally process-related).
 - **100-999:** General API design guidance
 
-### Google Product Areas
-
-- **2700-2799:** Apps (GSuite)
-- **2500-2599:** Cloud
-- **3000-3099:** Actions on Google
-- **4600-4699:** Geo
-
 To request a block for a specific team that is publishing API guidance or
 documentation germane to that specific team, reach out to api-editors@.
 
+### Custom AIPs
+
+If you creating your own set of AIPs for your company or organization and want
+to maintain compatibility with aip.dev, you will need to prefix your custom AIP
+IDs with a unique prefix. This will ensure that additional AIPs added to
+aip.dev will not conflict with any custom AIPs you might add.
+
+For example. assume you write a new AIP for state management in your AIPs, and
```

Comment:
We should:

- Drop the hyphen in our AIP referencing conventions (so `AIP 131` instead of `AIP-131` like we use now).
- Add the org prefix and hyphen for namespaced AIPs, e.g. `AIP ACU-9000`.

Or maybe we just leave the hyphen and have `AIP-ACU-9000`.

Reply:
Yeah it looks off. Happy to have something else. :-p